olive/engine/packaging/packaging_generator.py

```diff
@@ -170,10 +172,19 @@ def _upload_to_azureml_data(
     )
 
 
-def _get_model_info(node: "FootprintNode", model_rank: int):
+def _get_model_info(
+    node: "FootprintNode",
+    model_rank: int,
+    tempdir: Path,
+    model_path: Path,
+    output_dir: Path,
+    output_name: str,
+):
+    model_config = node.model_config
+    model_config["config"]["model_path"] = str(output_dir / output_name / model_path.relative_to(tempdir))
```

Comment:
>output_dir [](http://example.com/codeflow?start=47&length=10)

I am supposing the path is absolute path instead of relative path in zip file? 

It seems we need verify the path in UT.

Reply:
Basically, each PR need to update UT to make sure the feature complete before merging it.